public/app/features/variables/interval/actions.ts

```diff
@@ -29,22 +28,22 @@ export const updateIntervalVariableOptions = (
 };
 
 export interface UpdateAutoValueDependencies {
-  kbn: typeof kbn;
+  rangeUtil: any; // typeof rangeUtil?
```

Comment:
What about this?
```suggestion
  calculateInterval: typeof calculateInterval;
```

Reply:
I think I only use it as a dependency to simplify testing :)

Reply:
So you'll need to update some tests as well, but should be straightforward. Let me know if I can help out

Reply:
that worked!  thank you